### PR TITLE
Lpds 36 android card pattern

### PR DESCRIPTION
**Pull request by naufal22hadi**
# Description, Motivation and Context

`LPCard` is a card pattern component that can be changed based on the style. There are 5 style of `LPCard` :

1. Shadow (Default)
2. Solid Color
3. Banner Card
4. Border
5. List Card

Inside card component, there are 4 child component : 

1. Title (TextView)
2. Subtitle (TextView)
3. Content (TextView)
4. Button (Button)

# JIRA Ticket

[LPDS-36](https://lionparcel.atlassian.net/browse/LPDS-36)

# What is the destination branch of this PR?

Master

| Description | Image |
| -- | -- |
| Shadow | Screen Shot 2022-06-30 at 23 06 45 |
| Solid | Screen Shot 2022-06-30 at 23 07 09 |
| Banner | Screen Shot 2022-06-30 at 23 07 23 |
| Border | Screen Shot 2022-06-30 at 23 07 37 |
| List | Screen Shot 2022-06-30 at 23 07 53 |

### CODE REVIEW

File: sample/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -80,4 +80,11 @@
         android:layout_height="wrap_content"
         android:text="Empty State Component Sample"/>
 
+    <Button
+        android:id="@+id/button_to_cardSampleActivity"
```
Comment (fadhilahsaputro) [inline sample/src/main/res/layout/activity_main.xml:84]: mas nopal, ini naming convention buat id mau pake underscore atau capitalized ?

Comment (naufal22hadi) [reply sample/src/main/res/layout/activity_main.xml:84]: Screen Shot 2022-07-13 at 21 04 22

Sebetulnya gw cuma ngikutin yang sebelumnya aja sih mas, biasanya gw pake underscore sih mas. Enaknya diubah aja apa gimana mas?

Comment (falih-mulyana) [reply sample/src/main/res/layout/activity_main.xml:84]: kalau yang populer di layout XML itu pake underscore. Cuma di LP kita pakenya camelCase.

Comment (fadhilahsaputro) [reply sample/src/main/res/layout/activity_main.xml:84]: walah woke

**Review by fadhilahsaputro — APPROVED** [review]

File: sample/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (falih-mulyana) [reply sample/src/main/res/layout/activity_main.xml:84]: tapi gak masalah sih buat mas @naufal22hadi toh ini bukan CA. Mau buat standar sendiri gapapa asal konsisten.

Comment (naufal22hadi) [reply sample/src/main/res/layout/activity_main.xml:84]: oke2 makasih masukannya mas @fadhilahsaputro @falih-mulyana

**Pull request merged**